backend/config/curation-api.yml

```diff
@@ -729,84 +823,117 @@ components:
           nullable: true
           type: number
         title:
+          description: |
+            [title](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md#title)
           nullable: true
           type: string
         organism:
-          "$ref": "#/components/schemas/ontology_elements"
+          "$ref": "#/components/schemas/dataset_organism"
         processing_status:
           "$ref": "#/components/schemas/processing_status_dataset"
-        revised_at:
-          nullable: true
+        processing_status_detail:
           type: string
+          description: |
+            This field will be populated if an error that prevents the dataset from being fully processed.
```

Comment:
This field will be populated if an error ~that~ prevents the dataset from being fully processed.

Reply:
Also provided text in the other OpenAPI PR that would be specific to VALIDATION_FAILURE.